wled00/data/matrixtool/omggif.js

```
@@ -0,0 +1,809 @@
+// IN THE SOFTWARE.
+//
+// omggif is a JavaScript implementation of a GIF 89a encoder and decoder,
+// including animation and compression.  It does not rely on any specific
+// underlying system, so should run in the browser, Node, or Plask.
+
+"use strict";
+
+function GifWriter(buf, width, height, gopts) {
+  var p = 0;
+
+  var gopts = gopts === undefined ? { } : gopts;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'gopts'. Consider to delete it or rename it.

'gopts' is defined here:


(lint/suspicious/noRedeclare)